### ISSUE TITLE
Netflix doesn't show video when you continue watching 

### ISSUE THREAD

**Issue by srirambv** [brave/browser-laptop#11780]
### Description
Netflix doens't show video when you continue watching 

### Steps to Reproduce
  1. Visit Netflix, Click install for Widevine prompt, Allow auto-play in notification
  2. Login to Netflix, open any video under continue watching or open a new one
  3. Shows a blank black screen, video doesn't load or play
  4. Video overlay is not shown when you mouse over the player
  5. Click where the back button is, navigates back
  6. Click on the video again, starts playing on second click

**Actual result:**
Netflix doens't show video when you continue watching 

**Expected result:**
Should autoplay when you start the video the first time

**Reproduces how often:**
Consistently

### Brave Version

**about:brave info:**

Brave | 0.19.80
-- | --
rev | 7d07299
Muon | 4.5.13
libchromiumcontent | 62.0.3202.75
V8 | 6.2.414.36
Node.js | 7.9.0
Update Channel | Release
OS Platform | Microsoft Windows
OS Release | 10.0.15063
OS Architecture | x64

**Reproducible on current live release:**
Yes, quite annoying when you need to go back and reload the video to view

### Additional Information
Could be Windows specific, @LaurenWags @kjozwiak to verify on macOS
cc: @bbondy @bsclifton

**Comment by LaurenWags** [comment]
I don't recall seeing this earlier this week but I can definitely reproduce it now on MacOS.

**Comment by kjozwiak** [comment]
I can reproduce on macOS with `0.19.80` as well... Seems like it only happens when you initially install Brave... Once you've experienced the issue for the first time, I can't seem to reproduce it after the initial hiccup. If you remove cookies, you'll be able to reproduce it again once you've logged into Netflix.

**Comment by bsclifton** [comment]
@srirambv is this a duplicate of https://github.com/brave/browser-laptop/issues/11069 ?

**Comment by srirambv** [comment]
>@srirambv are you seeing this happen every single time you launch Netflix? Or does it just happen initially? 

Yes happens everytime for me.

>@srirambv is this a duplicate of #11069 ? (which I believe is also a dupe of #11270) 

Not exactly. I believe those issue happen all the time, Netflix issue happens only once. The second attempt onward works just fine.
